apps/coscrad-frontend/src/app/index-to-detail-flow-routes.tsx

```diff
@@ -17,20 +15,9 @@ export const IndexToDetailFlowRoutes = () => {
     const { indexToDetailFlows } = useContext(ConfigurableContentContext);
 
     return indexToDetailFlows.map(({ categorizableType, detailViewType, indexFilter }) => {
-        /**
-         * TODO [https://www.pivotaltracker.com/story/show/184069659]
-         * Fix this assymetry- it prevents our filtered index approach for notes
-         */
-        if (categorizableType === CategorizableType.note) {
-            return (
-                <>
-                    <Route path={routes.notes.index} element={<NoteIndexContainer />} />
-                    <Route path={routes.notes.detail()} element={<NoteDetailContainer />} />
-                </>
-            );
-        }
-
-        const routeBuilder = routes.resources.ofType(categorizableType);
+        const routeBuilder = isResourceType(categorizableType)
+            ? routes.resources.ofType(categorizableType)
```

Comment:
We may want a single `routes.forAggregate.ofType(aggregateType)` at some point to keep things polymorphic.

Reply:
That would help with [this story](https://www.pivotaltracker.com/story/show/184098960).